source/common/stats/symbol_table_impl.cc

```diff
@@ -41,19 +42,19 @@ uint64_t StatName::dataSize() const {
 #ifndef ENVOY_CONFIG_COVERAGE
 void StatName::debugPrint() {
   if (size_and_data_ == nullptr) {
-    ENVOY_LOG_MISC(info, "Null StatName");
+    std::cerr << "Null StatName" << std::endl;
```

Comment:
I prefer to use std::cerr so that I can call this from the debugger and see the results without restarting the binary with a differing logging level.

Reply:
nit: should this possibly be wrapped in a different LOG macro? Don't feel strongly about this.

Reply:
Sure that's fine.